.github/workflows/release.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         name: Set up Go
         uses: actions/setup-go@v3
         with:
-          go-version: 1.18
+          go-version: 1.20
```

Comment:
Need to upgrade this because sdk use url.JoinPath which is in newer version

Reply:
I think you could bump this to 1.21. Does it use the latest patch version automatically?

Reply:
I think it apply the latest patch version

Reply:
Fixed